eng/pipelines/installer/jobs/steps/build-linux-package.yml

```diff
@@ -8,18 +7,6 @@ parameters:
   subsetArg: ''
 
 steps:
-- ${{ if eq(parameters.buildTraversalBuildDependencies, true) }}:
-  - script: |
-      set -x
-      df -h
-      $(DockerRunMSBuild) ${{ parameters.image }} $(MSBuildScript) \
-        --ci \
-        /root/runtime/tools-local/tasks/installer.tasks/installer.tasks.csproj \
-        /t:Restore /t:Build /t:CreateHostMachineInfoFile \
```

Comment:
This was also doing `/t:Build` of the `installer.tasks.csproj`.  Isn't that part still needed?

Reply:
installer.tasks.csproj is implictly built when invoking build.cmd/sh: https://github.com/dotnet/runtime/blob/7cbf0a7011813cb84c6c858ef19acb770daa777e/Build.proj#L40-L60. This was probably a left-over when core-setup changed to repo tasks and isn't needed anymore.